cfg/mps/default_mps.yaml

```diff
@@ -12,82 +12,82 @@ llsfrb:
         type: BS
         host: localhost
         port: !tcp-port 4840
-        connection: plc
+        connection: mockup
```

Comment:
is it intended that the default is mockup?

Reply:
oh i did not notice :) i guess it makes sense , though